src/source/PeerConnection/PeerConnection.c

```diff
@@ -546,8 +557,16 @@ VOID onDtlsStateChange(UINT64 customData, RTC_DTLS_TRANSPORT_STATE newDtlsState)
 
     pKvsPeerConnection = (PKvsPeerConnection) customData;
 
-    if (newDtlsState == CLOSED) {
-        changePeerConnectionState(pKvsPeerConnection, RTC_PEER_CONNECTION_STATE_CLOSED);
+    switch (newDtlsState) {
+        case CONNECTED:
```

Comment:
BTW, can we change these enums PLEASE. We need to prefix these with RTC_DTLS_TRANSPORT_STATE like

RTC_DTLS_TRANSPORT_STATE_CONNECTED

I am not sure how we pushed this in. There are real chances for naming collisions. This is an internal API and we can change those just fine.

Reply:
Addressed